apps/storybook/src/TiledHeatmap.stories.tsx

```diff
@@ -116,23 +116,28 @@ class MandelbrotTilesApi extends TilesApi {
     return this.store.get({ layer, offset });
   }
 }
+
+const X_DOMAIN: Domain = [-2, 1];
+const Y_DOMAIN: Domain = [-1.5, 1.5];
```

Comment:
Alternative was to use `api.xDomain` and `api.yDomain` from the `MandelbrotTilesApi`, but it requires to override the props interface provided for the `Template`.

Reply:
I don't understand. You can use `api.xDomain` instead of `X_DOMAIN` inside the `Template` without changing the `props`, right?

Also, perhaps it is worth having a different story for non-pixel domain.

Reply:
> Template without changing the props, right?
Apparently no because of the typing `api.xDomain` is not part of `TilesApi`

Reply:
> Apparently no because of the typing `api.xDomain` is not part of `TilesApi`

Oh yes true. Didn't realize that `xDomain` and `yDomain` were implemented by `MandelbrotTilesApi`